### ISSUE TITLE
dependency resolution goes down inside unspecified items

### ISSUE THREAD

**Issue by grahamdyson** [DevSnicket/eunice#55]
Dependency resolution will consider descendants inside items that weren't specified in the dependency. This behaviour can result in misleading dependency counts that might be better not represented at all.

The following YAML will show a dependency even though "not parent" wasn't specified in the dependency on to child:
```yaml
- id: dependent
  dependsUpon: child
- id: not parent
  items: child
```

**Comment by grahamdyson** [comment]
released in [JavaScript 6.12.0](https://devsnicket.com/eunice/javascript/releases#6.12.0)

**Comment by grahamdyson** [comment]
devsnicket/eunice-dependency-and-structure@d8253978dfb24e0e9fa19ecfdc2294a449d46d5d
